codeworld-api/codeworld-api.cabal

```diff
@@ -36,6 +40,12 @@ Library
                        cereal               >= 0.5.4 && < 0.6,
                        cereal-text          >= 0.1.0 && < 0.2
 
+  if flag(reflex)
```

Comment:
Do you think a flag is the right answer here?  What exactly is the goal?

- If it's the additional dependency, I don't have any concerns about it.  I'm happy to just depend on reflex without the flag.
- If you're trying to dissuade usage until you're more confident in the API, consider a WARNING pragma instead?  Then people could still try it out on the live site.
- If it's about polluting the API, consider a different public module, like `CodeWorld.Reactive`, instead?

Or is there some reason we should be worried about pushing this live?